aibolit/patterns/multiple_while/multiple_while.py

```diff
@@ -1,13 +1,23 @@
 from aibolit.utils.ast_builder import build_ast
 from aibolit.ast_framework import AST, ASTNodeType
+from typing import List, Set
 
 
 class MultipleWhile:
 
     def __init__(self):
         pass
 
-    def value(self, filename: str):
+    def get_number_of_sequential_while_statement_in_function(self, tree: AST, node: int) -> int:
```

Comment:
Maybe call it `get_top_level_while_qty`?

Reply:
ok